File: NEsper/view/EventStream.cs
using System;

using net.esper.events;

namespace net.esper.view
{
	/// <summary>
    /// A streams is a conduct for incoming events. Incoming data is placed into streams for consumption by queries.
    /// </summary>

    public interface EventStream : Viewable
    {
        /// <summary> Insert a new event onto the stream.</summary>
        /// <param name="event">to insert
        /// </param>

        void Insert(EventBean _event);
    }
}

File: trunk/esper/src/main/java/com/espertech/esper/view/EventStream.java
package com.espertech.esper.view;

import com.espertech.esper.event.EventBean;

/**
 * A streams is a conduct for incoming events. Incoming data is placed into streams for consumption by queries.
 */
public interface EventStream extends Viewable
{
    /**
     * Insert a new event onto the stream.
     * @param event to insert
     */
    public void insert(EventBean event);
}
